src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "GrpcService-CSharp": {
       "commandName": "Project",
       "launchBrowser": false,
-      "applicationUrl": "https://localhost:5001",
+      "applicationUrl": "https://localhost:5001;http://localhost:50051",
```

Comment:
Why 50051? I'd prefer `http://localhost:5000` to be consistent with other ASP.NET apps.

Reply:
1. Consistent with other gRPC default endpoints. Pretty much all other gRPC tutorials use this port.
2. People are used to hitting port 500 with their browser for ASP.NET applications and it won't work in this case.

I don't feel strongly enough and could be convinced otherwise

Reply:
Will you convince yourself 😋 https://github.com/dotnet/aspnetcore/pull/11473/files#r297004913

It seems random that we use our default HTTPS endpoint but not HTTP endpoint. There is nothing special about that port number. I'd rather be consistent with ourselves.

I also don't want to update docs where I referred to port 5000 😄  https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-3.1#call-insecure-grpc-services-with-net-core-client

Reply:
Hey! In all fairness that screenshot has https as the scheme as not http 😄 

> I also don't want to update docs where I referred to port 5000 😄

That's reason enough for me